auth/jwt/token_test.go

```diff
@@ -106,6 +142,19 @@ func TestParse(t *testing.T) {
 
 	inValidToken := newToken("invalid", key())
 
+	createKeyAndToken := func(domain, subject string) (auth.Key, string) {
```

Comment:
No need for this function refactor `newToken` to provide a token when the subject or domain is empty